client/my-sites/customer-home/locations/secondary/learn-grow/index.jsx

```diff
@@ -2,7 +2,7 @@
  * External dependencies
  */
 import React, { useEffect } from 'react';
-import { connect } from 'react-redux';
+import { connect, useSelector } from 'react-redux';
```

Comment:
As a rule of thumb, it's a bit suspicious if one module uses both `connect` and the hook API. The component could be easily rewritten to avoid `connect` altogether:
```js
const dispatch = useDispatch();
const siteId = useSelector( getSelectedSiteId );
const cards = useLearnGrowCards( siteId );

useEffect( () => {
  if ( cards ) {
    dispatch( trackCardImpressions( cards ) );
  }
}, [ cards, dispatch ] );
```